src/dc_configure/auto_moz.rs

```diff
@@ -0,0 +1,224 @@
+use quick_xml;
```

Comment:
Lets call the module `configure`, give we touch it so much :)

Reply:
and the files `mozilla`/`outlook` or `auto_mozilla` and `auto_outlook`, not a big fan of these super shortened names (but that is mostly a personal preference)